_layouts/default.html

```diff
@@ -136,13 +136,14 @@ <h2 class="join-us-title">Join us on <a href="https://github.com/a11yproject/a11
             <li><a class="join-us-btn" href="https://github.com/a11yproject/a11yproject.com/issues"><span class="data open-issues">&nbsp;</span>Open Issues</a></li>
           </ul>
 
-          <ul id="contributors-list" class="row"></ul>
+          <ul class="contributors-list"></ul>
           <ul class="pager contributors-pagination inline-list"></ul>
         </div>
 
         <div class="colophon">
-
           <h5 class="screen-reader-text">Colophon</h5>
+
+          <p class="no-lead">Stay in touch</p>
```

Comment:
In terms of the phrase I used here... does anyone have thoughts on adding this bit of text (of if it should say something else)?

Reply:
this is fine.  let's not add anymore beyond this though.